gluetool/tool.py

```diff
@@ -220,7 +220,8 @@ def _handle_failure_core(self, failure):
 
         logger.error(msg, exc_info=failure.exc_info)
 
-        if self.sentry is not None:
+        # Submit what hasn't been submitted yet...
+        if self.sentry is not None and failure.sentry_event_id is None:
```

Comment:
Why not use `if self.sentry and not failure.sentry_event_id:`? I think that could improve readability

Reply:
Right. Fixed.